apis/reaper/v1alpha1/reaper_types.go

```diff
@@ -155,6 +136,30 @@ type AutoScheduling struct {
 	ExcludedKeyspaces []string `json:"excludedKeyspaces,omitempty"`
 }
 
+type ReaperClusterTemplate struct {
+	ReaperDatacenterTemplate `json:",inline"`
+
+	// The keyspace to use to store Reaper's state. Will default to "reaper_db" if unspecified. Will be created if it
+	// does not exist, and if this Reaper resource is managed by K8ssandra.
+	// +kubebuilder:default="reaper_db"
+	// +optional
+	Keyspace string `json:"keyspace,omitempty"`
+
+	// Defines the username and password that Reaper will use to authenticate CQL connections to Cassandra
+	// clusters. These credentials need to be stored on each Cassandra node. If CQL authentication is not required,
```

Comment:
Actually this sentence is not accurate: the CQL user secret does not need to be "stored" on nodes, however the secret needs to be created beforehand so that the corresponding role can be created by cass-operator. I will fix the comment.